app/visualize/analysis/stmt/expr/expr_traveler.py

```diff
@@ -67,6 +72,51 @@ def _list_travel(node: ast.List, elem_manager: CodeElementManager):
         elts = [ExprTraveler.travel(elt, elem_manager) for elt in node.elts]
         return ListExpr.parse(elts)
 
+    def compare_travel(node: ast.Compare, elem_manager: CodeElementManager):
+        if isinstance(node, ast.Compare):
+            left = ExprTraveler.compare_travel(node.left, elem_manager)
+            comparators = [ExprTraveler.compare_travel(comparor, elem_manager) for comparor in node.comparators]
+
+            return CompareExpr.parse(left, tuple(comparators), tuple(node.ops))
+
+        elif isinstance(node, ast.BinOp):
+            return ExprTraveler.binop_travel(node, elem_manager)
+
+        elif isinstance(node, ast.Name):
+            return ExprTraveler.name_travel(node, elem_manager)
+
+        elif isinstance(node, ast.Constant):
+            return ExprTraveler.constant_travel(node)
+
+    @staticmethod
+    def _get_func_name(node: ast, elem_manager: CodeElementManager):
+        if isinstance(node, ast.Name):
+            return node.id
+
+        elif isinstance(node, ast.Attribute):
+            raise NotImplementedError(f"[call_travel] {type(node)}정의되지 않았습니다.")
+
+        else:
+            raise TypeError(f"[call_travel] {type(node)}는 잘못된 타입입니다.")
+
+    @staticmethod
+    def _compare_travel(node: ast.Compare, elem_manager: CodeElementManager):
+        if isinstance(node, ast.Compare):
+            left = ExprTraveler._compare_travel(node.left, elem_manager)
+            comparators = [ExprTraveler._compare_travel(comparor, elem_manager) for comparor in node.comparators]
+
+            return CompareExpr.parse(left, tuple(comparators), tuple(node.ops))
```

Comment:
배열로 저장하고 tuple로 다시 만들어주는 이유가 따로 있나요? 바로 tuple로 만들어서 넣어줘도 괜찮을 것 같아요!

Reply:
아마 바로 위 코드에서 요소가 계속 추가되기 때문에 수정이 불가능한 튜플로는 추가가 안되지 않을까요? 하는 방법이 따로 있을까요?

Reply:
```suggestion
            comparators = tuple(ExprTraveler._compare_travel(comparor, elem_manager) for comparor in node.comparators)

            return CompareExpr.parse(left, comparators, tuple(node.ops))
```

이렇게 바로 만들 수 있지 않을까요?

Reply:
오 이렇게도 쓸 수 있군요! 수정했습니다.